### ISSUE TITLE
Collaborate?

### ISSUE THREAD

**Issue by alphapapa** [paul-issartel/nnreddit#5]
BTW, do you know about https://github.com/fourier/gnus-reddit/issues/1#issuecomment-212535851 ?  Maybe you two could collaborate.  :)

**Comment by paul-issartel** [comment]
Yeah, I stumbled upon this project when I first looked up existing backends, but I seemed incomplete. However, it was very helpful as an example of how to write a new Gnus backend.

Also, it looks like someone else forked my code and is adding a bunch of new features:
https://github.com/rprospero/nnreddit/tree/oauth

It's great to see how quickly things can evolve with free software :)

**Comment by alphapapa** [comment]
Awesome.  :D

**Comment by alphapapa** [comment]
Ah, too bad he hasn't enabled issues for his fork, so we can't talk about these things there.  :(  I'll see if I can send him a reddit message.

**Comment by rprospero** [comment]
Well, never expected my little fork to get noticed.  I've turned on issues and I'll be happy to help, though I'll also freely admit that A) I'm pretty terrible at elisp and B) I already have another open source project that I'm supposed to be helping and am currently contributing jack.

That said, I currently have some rudimentary oauth working for logging in and a crummy little mode for reading private messages.  My intent was to add that into the Gnus code, but I haven't quite figured out how all of that works yet.

**Comment by alphapapa** [comment]
> My intent was to add that into the Gnus code, but I haven't quite figured out how all of that works yet.

Can't blame you there, haha.  :D

**Comment by tzz** [comment]
@rprospero @paul-issartel would you be interested to get this with or without the oauth stuff into Gnus itself (by merging into Emacs, Gnus is no longer a separate Git repo)? You can still keep the private repos on GitHub, as long as the license is compatible and you're willing to assign the rights to the Emacs project. Or you can make it a GNU ELPA package. Either way it would be available to all users.
